### PR TITLE
Added creating, render and styles

### PR DESCRIPTION
**Pull request by MkpDolganovKirill**

### CODE REVIEW

File: src/components/Create.jsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    };
+  };
+
+  const checkEnter = (event) => {
+    if (event.code === "Enter") createTask();
+  };
+
+  return (
+    <div className="create-component">
+      <h1 className="title">TO-DO LIST</h1>
```
Comment (DoctorFreed) [inline src/components/Create.jsx:27]: move to app.js

Comment (omanei) [inline src/components/Create.jsx]: move to app.js

File: src/App.js [hunk 2]
```diff
@@ -25,6 +25,18 @@ const App = () => {
     });
   };
 
+  const deleteTask = async (taskId) => {
+    const newTasks = [...tasks];
+    await axios.delete(`http://localhost:8000/deleteTask?_id=${taskId}`).then(res => { });
+
+    newTasks.forEach((el, index) => {
```
Comment (omanei) [inline src/App.js]: get update in from db

File: src/App.js [hunk 3]
```diff
@@ -25,6 +25,18 @@ const App = () => {
     });
   };
 
+  const deleteTask = async (taskId) => {
```
Comment (omanei) [inline src/App.js]: put this in task components

File: src/App.js [hunk 4]
```diff
@@ -25,6 +25,20 @@ const App = () => {
     });
   };
 
+  const editTask = async (editingTask) => {
+    const newTasks = [...tasks];
+    const isCheck = editingTask.isCheck;
+    const updateTask = { ...editingTask, isCheck: !isCheck }
+    await axios.patch('http://localhost:8000/updateTask', updateTask).then(res => { });
+
+    newTasks.forEach((el, index) => {
```
Comment (omanei) [inline src/App.js]: get update in from db

File: src/components/Tasks.jsx [hunk 5]
```diff
@@ -1,46 +1,19 @@
...
-              <img
-                className={isCheck ? 'trash-icon-complete' : 'trash-icon'}
-                src={trash} alt='delete'
-                onClick={() => remove(_id)}
-              />
-            </div>
-          );
-        })
+        allTasks.map((task, index) => <Task 
+          key={task._id}
```
Comment (DoctorFreed) [inline src/components/Tasks.jsx:11]: you already send full task

**Pull request merged**